src/htm/engine/RESTapi.cpp

```diff
@@ -24,6 +24,7 @@ Implementation of the RESTapi class
 #include <htm/engine/Network.hpp>
 
 #define RESOURCE_TIMEOUT 86400    // one day
+#define ID_MAX=9999               // maximum number of generated ids  (this is arbitrary)
```

Comment:
I think this is invalid syntax, best would be avoid define and `const size_t ID_MAX=9999;`